### PR TITLE
6pm 1 office hours table

### PR DESCRIPTION
**Pull request by carsoncoley**
Make office hours table rows alternate color; 6pm 1 office hours table

### CODE REVIEW

File: javascript/src/main/components/OfficeHours/OfficeHourTable.js [hunk 1]
```diff
@@ -37,14 +37,14 @@ export default ({officeHours,admin,deleteOfficeHour}) => {
 
     if (admin) {
         columns.push({
-            text: "Delete",
+            text:  "Delete",
```
Comment (LeoJr99) [inline javascript/src/main/components/OfficeHours/OfficeHourTable.js:40]: Everything looks good. Maybe delete the white space in line 40 to reduce the amount of changes made

**Review by LeoJr99 — APPROVED** [review]

**Review by mara-downing — CHANGES_REQUESTED** [review]
I agree with Leo, everything looks good but you should delete the space before "Delete" for consistency.

File: javascript/src/main/components/OfficeHours/OfficeHourTable.js [hunk 2]
```diff
@@ -37,14 +37,14 @@ export default ({officeHours,admin,deleteOfficeHour}) => {
...
             isDummyField: true,
             dataField: "delete",
             formatter: (_cell, row) => renderDeleteButton(row.id)
         });
     }
 
     return (
-        <BootstrapTable keyField='id' data={officeHours} columns={columns} />
+        <BootstrapTable keyField='id' data={officeHours} columns={columns} striped={true}/>
```
Comment (btk5h) [inline javascript/src/main/components/OfficeHours/OfficeHourTable.js:48]: The `={true}` is redundant in React. If you change this to `striped`, React will implicitly treat this as `striped={true}`.
```suggestion
        <BootstrapTable keyField='id' data={officeHours} columns={columns} striped />
```

**Review by btk5h — CHANGES_REQUESTED** [review]

**Review by LeoJr99 — APPROVED** [review]
LGTM

**Review by LeoJr99 — APPROVED** [review]
LGTM

**Review by btk5h — APPROVED** [review]

**Pull request merged**